contents/code/tilingmanager.js

```diff
@@ -447,7 +447,7 @@ function TilingManager(timer) {
                               });
         KWin.registerShortcut("TILING: Focus next tile",
                               "Focus next tile",
-                              "Meta+]",
+                              "",
```

Comment:
Are you sure about this one and the next? It might be just about my workflow, but I consider them to be quite essential.

Reply:
KWin already has shortcuts to select the window above/below/to the right/left.

These are only really of use if you care about the tile *order* in a numeric sense.